challenge-13/submissions/atplay90/solution-template.go

```
@@ -0,0 +1,201 @@
+
+
+// InitDB sets up a new SQLite database and creates the products table
+func InitDB(dbPath string) (*sql.DB, error) {
+	// Open a SQLite database connection
+	db, err := sql.Open("sqlite3", dbPath)
+    if err != nil {
+        return nil, err
+    }
+    
+    // Test the connection
+    if err = db.Ping(); err != nil {
```

golangci-lint (2.12.2)

[error] (*database/sql.DB).Ping must not be called. use (*database/sql.DB).PingContext

(noctx)